database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowStoreEntryComponent.scala

```diff
@@ -106,4 +131,14 @@ trait WorkflowStoreEntryComponent {
       if workflowStoreEntry.workflowExecutionUuid === workflowId
     } yield workflowStoreEntry.workflowState
   )
+
+  /**
+    * Useful for updating a given workflow to a new state
```

Comment:
Really?

Reply:
Blah I was just copy pasting it from other queries :p